_guides/responsive/control_layout.md

```diff
@@ -155,6 +141,12 @@ In this simple example,
 </amp-img>
 {% endhighlight %}
 
+**Note:** AMP supports the `w` descriptor across all browsers;
+however, Safara doesn't support it natively.
```

Comment:
Safari not Safara.

No need to mention it specifically. The important message is that it works everywhere.